src/sentry/api/permissions.py

```diff
@@ -81,6 +81,11 @@ def has_permission(self, request, *args, **kwargs):
     def has_object_permission(self, request, *args, **kwargs):
         return super().has_object_permission(request, *args, **kwargs) or is_active_staff(request)
 
+    def is_not_2fa_compliant(self, request, *args, **kwargs) -> bool:
```

Comment:
Superuser already skips this check in `OrganizationPermission`'s `is_not_2fa_compliant`.

Staff needs to also do this in order to access the org details endpoint when 2FA is required.